components/ws-daemon/pkg/cgroup/plugin_iolimit_v2.go

```diff
@@ -61,38 +64,52 @@ func (c *IOLimiterV2) Apply(ctx context.Context, basePath, cgroupPath string) er
 	return nil
 }
 
-func (c *IOLimiterV2) writeIOMax(loc string) error {
-	iostat, err := os.ReadFile(filepath.Join(string(loc), "io.stat"))
+func (c *IOLimiterV2) writeIOMax(cgroupPath string) error {
+	iostat, err := os.ReadFile(filepath.Join(string(cgroupPath), "io.stat"))
 	if os.IsNotExist(err) {
+		log.Error("Pod is gone")
```

Comment:
I feel it isn't an error.  Timing issues may arise as per the comment below.
https://github.com/gitpod-io/gitpod/pull/9309/files#diff-9924bdc24dd06931f08b754cc11839d09c8ad010ce1fb4cc2c376c8b752b0538R71

Reply:
removed